shell/platform/android/io/flutter/plugin/editing/TextInputPlugin.java

```diff
@@ -791,31 +852,60 @@ public void autofill(SparseArray<AutofillValue> values) {
       return;
     }
 
-    final HashMap<String, TextInputChannel.TextEditState> editingValues = new HashMap<>();
-    for (int i = 0; i < values.size(); i++) {
-      int virtualId = values.keyAt(i);
+    if (configuration.enableDeltaModel) {
```

Comment:
I thought about this, I had included it just because I was trying to get every feature that works with the TextField on the engine side working with this delta model. I agree rich text editors will probably not be using autofill, do you think it is alright to not include this then? I was thinking this might be good to have for the future but I guess are plans for this are not concrete.

Reply:
I think the thing autofill does is slightly different from any of the delta types specified: it replaces the entirety of the document, instead of the content in  `[0, currentLength)`. They're slightly different since we're dealing with async communications. 

For example if the user autofills a password field and at the same time the framework changes the text to be longer, then we want the obscured field has the right characters in it, instead of just replacing the original range.